### ISSUE TITLE
[Question] Can we set different paths for routes in nuxt 3 with this package?

### ISSUE THREAD

**Issue by vince83110** [intlify/nuxt3#40]
With Nuxt2, it was easy to define paths for each locale (https://i18n.nuxtjs.org/routing)

Is it possible using this package? I can't see in the doc, and I looked at the code quickly but didn't find anything...

Thanks !

**Comment by vflorian2000** [comment]
Seems like routing is not implemented at all. Browser language detection doesn't work for me either.

**Comment by vince83110** [comment]
Yes weird... I can't understand how we can talk about i18n and not talk about paths .. ? I can't imagine how much work it's necessary here, but, how can I have my Nuxt 3 app declined for my locales ? Should I drop the idea it's possible ?

**Comment by nikita-komissarov** [comment]
+1 How do you use this without localizing the routes?

**Comment by NidMo** [comment]
Does this package implement this function ？

**Comment by b-arabah** [comment]
+1

Will this package implement this function ？
